packages/model-viewer/src/features/ar.ts

```diff
@@ -72,6 +73,8 @@ export const ARMixin = <T extends Constructor<ModelViewerElementBase>>(
     ModelViewerElement: T): Constructor<ARInterface>&T => {
   class ARModelViewerElement extends ModelViewerElement {
     @property({type: Boolean, attribute: 'ar'}) ar: boolean = false;
+    @property({type: Boolean, attribute: 'ar-autogenerate-usdz'})
```

Comment:
Did you see my comment about doing this without introducing a new attribute? If you look at my PR, I changed the logic a bit so that the presence of `ios-src` automatically enables Quick Look, while explicitly adding the `quick-look` `ar-mode` opts in to USDZ generation. I think I prefer that approach; what do you think? Would you mind merging in that part of my PR here?